app/controllers/tours_controller.rb

```diff
@@ -30,6 +30,11 @@ def show
     @tour = current_staff_user.tours.find(params[:id])
   end
 
+  def destroy
+    Tour.find(params[:id]).destroy
+    render :index
```

Comment:
This should not render the index. Usually, the response for the `destroy` action is `head :no_content`, which is an empty response.